Client/game_sa/CGameSA.cpp

```diff
@@ -125,7 +125,7 @@ CGameSA::CGameSA()
     m_pControllerConfigManager = new CControllerConfigManagerSA();
     m_pProjectileInfo = new CProjectileInfoSA();
     m_pRenderWare = new CRenderWareSA();
-    m_pHandlingManager = new CHandlingManagerSA();
+    m_HandlingManager = std::make_shared<CHandlingManagerSA>();
```

Comment:
why make_shared and not make_unique?

Reply:
> why make_shared and not make_unique?

For me, it was an experiment, which turned out to be unsuccessful. I will fix this in the next commit, as mentioned above.

Reply:
Then please don't mark as resolved until it's done

Reply:
> Then please don't mark as resolved until it's done

This duplicates what is already above, @TracerDS. Can I close this?

Reply:
No need to ask. Sure, close this, its resolved.